splitio/recorder/recorder.py

```diff
@@ -131,22 +136,23 @@ def record_treatment_stats(self, impressions, latency, operation):
             impressions = self._impressions_manager.process_impressions(impressions)
             if not impressions:
                 return
-            # pipe = self._make_pipe()
-            # self._impression_storage.add_impressions_to_pipe(impressions, pipe)
-            # self._telemetry_storage.add_latency_to_pipe(operation, latency, pipe)
-            # result = pipe.execute()
-            # if len(result) == 2:
-            #   self._impression_storage.expire_key(result[0], len(impressions))
-            self._impression_storage.put(impressions)
+            pipe = self._make_pipe()
+            self._impression_storage.add_impressions_to_pipe(impressions, pipe)
+            result = pipe.execute()
+            if len(result) == 2:
+                self._impression_storage.expire_key(result[0], len(impressions))
+            if method_name is not None:
+                self._telemetry_redis_storage.record_latency(method_name[4:], latency)
         except Exception:  # pylint: disable=broad-except
             _LOGGER.error('Error recording impressions')
             _LOGGER.debug('Error: ', exc_info=True)
 
-    def record_track_stats(self, event):
+    def record_track_stats(self, event, latency):
         """
         Record stats for tracking events.
 
         :param event: events tracked
         :type event: splitio.models.events.EventWrapper
         """
+        self._telemetry_redis_storage.record_latency(MethodExceptionsAndLatencies.TRACK.value, latency)
```

Comment:
this should be also pipelined

Reply:
Yep, I added the pipe in the RedisTelemetryStorage class